partner_company_type/models/res_partner.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2017-2018 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    partner_company_type_id = fields.Many2one(
+        comodel_name='res.partner.company.type',
+        string='Partner Company Type',
```

Comment:
@GillesTephaneMeyomesse As discussed, s/Partner Company Type/Legal Form/

Reply:
@sbidoul seems like it's fixed. should be good to go.